CAFAna/Probabilities/plot_biprobability.C

```diff
@@ -0,0 +1,684 @@
+//////////////////////////////////////////////////////////////////
```

Comment:
Can we make the directory name probabilities/ with a lowercase p? The convention is that the capital directories are CAFAna source files, and the lowercase ones are macros.